src/components/players/player/styles.ts

```diff
@@ -1,13 +1,55 @@
 import { type SxProps } from "@mui/material";
+import { theme } from "~/styles/theme.js";
 
 export const styles = {
   container: {
-    flexDirection: "row",
+    position: "relative",
+    flexDirection: "column",
     alignItems: "center",
     gap: 1,
   },
-
+  character: {
+    textTransform: "uppercase",
+  },
+  selectBot: {
+    textTransform: "uppercase",
+  },
+  avatarContainer: {
+    position: "absolute",
+  },
+  avatarResult: {
+    zIndex: 1,
+  },
   avatar: {
+    width: 150,
+    height: 150,
+    "& > img": {
+      width: "inherit",
+      height: "inherit",
+    },
     cursor: "pointer",
   } satisfies SxProps,
+  botResult: (stage: string, isBot?: boolean, isSelected?: boolean) => {
+    let textColor = theme.palette.common.white;
+
+    if (stage === "voting") {
+      textColor = isSelected
+        ? theme.palette.success.main
+        : theme.palette.common.white;
+    } else if (stage === "results") {
+      if (isSelected === false && isBot === false) {
```

Comment:
You don't need to explicitly declare the equality check (`===`) since it's already a boolean. 
You can just `!isSelected && !isBot`